pkg/metrics/prometheus.go

```diff
@@ -37,20 +38,33 @@ type Collector interface {
 	RecordAPISIXLatency(time.Duration)
 	// IncrAPISIXRequest increases the number of requests to apisix.
 	IncrAPISIXRequest(string)
+	// increases the number of cluster health check operations with the controller
+	// name label
+	IncrCheckClusterHealth(string)
+	// IncrSyncSuccess increases the number of sync operations with the resource
+	// type label
+	IncrSyncSuccess(string)
```

Comment:
It's not a good name, we may use `IncrSyncOperation`, and add another label like `success` to mark whether the sync operation is successful or not. Then we also can increase the failure counts.

Reply:
resolved.
now sync_operation_total has a label `state` to mark the sync operation is successful or raises error